corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -448,9 +449,11 @@ hqDefine('app_manager/js/releases/releases', function () {
                         self.addSavedApp(app, true);
                     }
                     self.buildState('');
+                    self.buildStatusCode(200);
```

Comment:
may be obvious, whats the reason for setting initial value to 200? Can it be left blank only?

Reply:
I was debating whether it'd be clearer to use 200 (HTTP "OK" code) or just blank for a non-error state. Updated to use blank.